README.md

```diff
@@ -148,6 +149,10 @@ Both `BaseCNN` and `BaseLSTM` also have prediction explanation mechanisms that c
 If you have ideas on how to improve the framework to assess text conversation for constructive and inclusive
 communication, we welcome your contributions!
 
+## Running tests
+```python
+python3 -m unittest ml-conversational-analytic-tool/tests/<test_file>.py
```

Comment:
So this has to be manually run for each unit test file? How about an automated test suite runner?

Reply:
we can use `python3 -m unittest discover -s tests` to run all the tests in one go. Did you mean to say we need to use something like tox when you refer "automated test runner"?

Reply:
The example given in your README change states how to test file-by-file, not as a collection of tests. Tox isn't a requirement, but it can be effective.

Reply:
I think it would be helpful to include both
- a command for executing all tests
- a command for executing one specific file

and you shouldn't need to specify `python3` here as that's already defined in the virtualenv

Reply:
Sounds reasonable. Will do update the README.md for both!